File: apprefurbished/src/main/java/com/androidcollider/easyfin/common/managers/api/ApiModule.java
package com.androidcollider.easyfin.common.managers.api;

import dagger.Module;
import dagger.Provides;

/**
 * @author Ihor Bilous
 */

@Module
public class ApiModule {

    @Provides
    ApiManager provideApiManager() {
        return new ApiManager();
    }
}

File: apprefurbished/src/main/java/com/androidcollider/easyfin/common/managers/api/ApiModule.kt
package com.androidcollider.easyfin.common.managers.api

import dagger.Module
import dagger.Provides

/**
 * @author Ihor Bilous
 */
@Module
class ApiModule {
    @Provides
    fun provideApiManager(): ApiManager {
        return ApiManager()
    }
}